app/controller/List.js

```diff
@@ -1044,17 +1075,24 @@ Ext.define('WhatsFresh.controller.List', {
 	// Initialize functions
 	launch: function(){
 		this.callParent(arguments);
-
+		this.getDistanceSelect().disable();
 		// Variables
 			// FOR: back button functionality
 			WhatsFresh.pvalue = [];
 			WhatsFresh.path = [];
 			WhatsFresh.pcount = 0;
 			WhatsFresh.backFlag = 0;
-			// FOR: checkboxes 
+			// FOR: checkboxes
 			WhatsFresh.use = 1;
 			WhatsFresh.use2 = 1;
 			WhatsFresh.infowindowFlag = 0;
+			// FOR: user location printout
+			WhatsFresh.userLoc = 0;
+			WhatsFresh.dist = {
+				distance: "200 miles", 
+				value: 200, 
+				unit: "miles"
+			};
```

Comment:
Why does this need to be in the global scope?
This looks like a store record. If it's a store record, we should be using the record itself instead of duplicating it.